examples/python/ruff.toml

```diff
@@ -0,0 +1,228 @@
+# See https://docs.astral.sh/ruff/configuration/
+# For a full list of supported rules, see https://docs.astral.sh/ruff/rules/
+# For a full list of options, see https://docs.astral.sh/ruff/settings/
+
+target-version = "py39"
```

Comment:
Per chat discussion, let's move common item to the top-level ruff.toml and use https://docs.astral.sh/ruff/settings/#extend to